apps/site/next-data/downloadSnippets.ts

```diff
@@ -29,7 +29,7 @@ export default async function getDownloadSnippets(
     const { default: provideDownloadSnippets } = await import(
       '@/next-data/providers/downloadSnippets'
     );
-    return provideDownloadSnippets(lang)!;
+    return (await provideDownloadSnippets(lang))!;
```

Comment:
I don't like `!` assertions -- can we do something else? :)

Reply:
but I didn't add the `!` 🥹 , not to increase the PR's scope I'd be tempted to keep it as if here and address it in a separate PR?

Reply:
Gotcha, can we at least separate the await with the return with (!) 🙏

Reply:
change reverted, no longer needed (thanks to work done on the adapter 🙂) 🥳